soc/xtensa/intel_adsp/common/CMakeLists.txt

```diff
@@ -100,3 +100,5 @@ add_custom_target(
       ${CMAKE_BINARY_DIR}/zephyr/main.mod
       ${CMAKE_BINARY_DIR}/zephyr/main.mod 2>${NULL_FILE}
 )
+
+add_subdirectory(ace)
```

Comment:
Hmm, this is a bit strange. Why have a "ace" folder under "common" when there is a separate folder for this hardware already?

Reply:
Yes it is very strange and obviously wrong.
I am just not sure how much refactoring I can push here.
And it could probably affect also SOF upstream.
Maybe lets skip it for now?